vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItem.kt

```diff
@@ -115,6 +116,38 @@ abstract class RoomSummaryItem : VectorEpoxyModel<RoomSummaryItem.Holder>(R.layo
             it.performHapticFeedback(HapticFeedbackConstants.LONG_PRESS)
             itemLongClickListener?.onLongClick(it) ?: false
         }
+//        if (matrixItem.displayName!!.contains("[TG] ")) {
```

Comment:
I think the best is to keep minimal changes for avoiding conflicts with upstream